### PR TITLE
Request documentation

### PR DESCRIPTION
**Pull request by JulianGCalderon**
This PR adds documentation on different kind of requests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,3 +126,46 @@ After Comet BFT receives the transaction, its relayed to the application through
 
 The response is then encoded in the transaction result, and added to the blockchain.
 
+### Sending Formats
```
Comment (ManuelBilbao) [inline README.md:129]: Maybe we should say something like "Transaction sending formats"? To clarify what we are talking about

**Review by ManuelBilbao — COMMENTED** [review]
This is great! It'll help solving #15 
I leave you a little comment, nothing so important

File: README.md (hunk 1 above)
Comment (JulianGCalderon) [reply README.md:129]: I'm not sure about that name as a transaction is not the only message that can be sent to the node, but I agree that the section name is not really clear enough. how about "Interacting with a Node"? (inspired from the official docs https://docs.cosmos.network/main/user/run-node/interact-node#using-grpc)

Comment (JulianGCalderon) [reply README.md:129]: Something like this:

```suggestion
### Interacting with a Node

The full-node exposes three different types of endpoints for interacting with it.
```

Comment (ManuelBilbao) [reply README.md:129]: Like it!

**Review by ManuelBilbao — APPROVED** [review]

**Review by fborello-lambda — APPROVED** [review]
LGTM

**Review by jotabulacios — APPROVED** [review]
LGTM

**Pull request merged**